packages/karma/index.js

```diff
@@ -67,17 +67,15 @@ module.exports = (options = {}) => (neutrino) => {
             [tests]: ['webpack'],
             [sources]: ['webpack'],
           },
-          webpackMiddleware: {
-            // Only display webpack compile duration and errors/warnings, since
-            // the focus should be on the output from the tests/karma instead.
+          webpack: {
+            ...omit(neutrino.config.toConfig(), ['entry']),
```

Comment:
So far, since webpack 5 is dropping shims, this is one way to make the default template work with karma. 

```
neutrino.config
      .plugin('provide-shim')
      .use(require.resolve('webpack/lib/ProvidePlugin'), [{
        assert: require.resolve('assert'),
        process: require.resolve('process/browser'),
      }]);
```

Once I get the new webpack-chain, I'll be able to test the resolve.fallback new Mapping instead of providing a plugin. I also do not like this approach, to have to shim just what's needed within the karma package. Karma compiles ahead of time and does have access to the node runtime to get `assert`.

Reply:
Will add the above for now to make the CI tests pass.